db/migrate/20210622002941_add_team_id_to_consumer_apps.rb

```diff
@@ -0,0 +1,5 @@
+class AddTeamIdToConsumerApps < ActiveRecord::Migration[6.1]
+  def change
+    add_column :consumer_apps, :team_id, :string
```

Comment:
`team_id` is the new value we would store for Consumer Apps. Only needed for iOS Consumer Apps, so it's left as optional ("nullable")